plugins/newsql/newsql.c

```diff
@@ -1068,6 +1068,20 @@ static int newsql_param_value(struct sqlclntstate *clnt,
     int len = val->value.len;
     void *p = val->value.data;
 
+    if (val->int64_arr && val->int64_arr->n_elements) {
```

Comment:
Can we use `OneOf` in the proto definition and use a switch case here?

Reply:
Yep. Took a while to upgrade protoc-c but working now.